controllers/comment.controllers.js

```diff
@@ -0,0 +1,21 @@
+const { fetchArticleComments } = require("../models/comment.models.js");
+
+const getArticleComments = (request, response, next) => {
+    const { article_id } = request.params;
+
+    fetchArticleComments(article_id)
+        .then(({ rows: results, rowCount}) => {
```

Comment:
If the article exists, but it has no comments, we want to send back a 200 and just send back an empty array